config.toml

```diff
@@ -2,8 +2,8 @@ baseurl = "https://www.gopheracademy.com"
 languageCode = "en-us"
 title = "Gopher Academy | Home"
 contentdir = "content"
-layoutdir = "layouts-main"
-publishdir = "public-main"
+layoutdir = "layouts"
+publishdir = "public"
```

Comment:
I suggest removing those settings entirely. Those are the defaults,  and these will never change in Hugo.